bundler/spec/support/artifice/vcr_cassettes/realworld/index.rubygems.org/info/hoe-yard/GET/response

```diff
@@ -0,0 +1,34 @@
+HTTP/1.1 200 OK
+connection: keep-alive
+content-length: 384
+content-type: text/plain; charset=utf-8
+x-frame-options: SAMEORIGIN
+x-xss-protection: 1; mode=block
+x-content-type-options: nosniff
+x-download-options: noopen
+x-permitted-cross-domain-policies: none
+referrer-policy: strict-origin-when-cross-origin
+etag: "fcba05bc4488cd21065b564db5c0e047"
+last-modified: Mon, 14 Dec 2020 16:36:07 GMT
+cache-control: max-age=60, public
+content-security-policy: default-src 'self'; font-src 'self' https://fonts.gstatic.com; img-src 'self' https://secure.gaug.es https://gravatar.com https://secure.gravatar.com https://*.fastly-insights.com; object-src 'none'; script-src 'self' https://secure.gaug.es https://www.fastly-insights.com; style-src 'self' https://fonts.googleapis.com; connect-src 'self' https://s3-us-west-2.amazonaws.com/rubygems-dumps/ https://*.fastly-insights.com https://fastly-insights.com https://api.github.com; form-action 'self'; frame-ancestors 'self'
```

Comment:
Btw. I do wonder why we do set these for `index.rubygems.org`. I'll take a look.

Reply:
my guess is that it is probably a Fastly VCL header that is applied to all domains